ocs_ci/helpers/dr_helpers.py

```diff
@@ -166,7 +189,11 @@ def failover(
     logger.info(
         f"Wait for {constants.DRPC}: {drpc_obj.resource_name} to reach {constants.STATUS_FAILEDOVER} phase"
     )
-    drpc_obj.wait_for_phase(constants.STATUS_FAILEDOVER)
+    relocate_condition = constants.STATUS_FAILEDOVER
+    if discovered_apps:
+        relocate_condition = constants.STATUS_RELOCATING
```

Comment:
why are we waiting for Relocating status on Failover function?

Reply:
The next commit will remove this addition that was made by mistake.